storage/integration-test/src/storage-mq.test.js

```diff
@@ -114,6 +114,45 @@ describe('IT against Storage-MQ service', () => {
     expect(allContentResponse.body[0]).toEqual(contentResponse.body)
   }, TIMEOUT)
 
+  test('Should create bucket with array content', async () => {
```

Comment:
Would be nicer to refactor the tests to run the generic "create a pipeline, insert data, read data" without code duplication. I had a look at by writing a function that returns the configured test function itself that but ran into a jest warning that the test does not contain an assert.

Reply:
I think you can use jest's [`test.each(...)` function](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) for that.